### PR TITLE
Create Popup Modal

### PR DESCRIPTION
**Pull request by Ol-create**
- Create a data object.
- Create project array
- Dynamically render modal box with javascript function
- Create project title node
- Create language list node
- Create an Image node
- Create page description text node
- Add event listener to close modal
- Append modal to the body element
- Add event listener to show popup window

### CODE REVIEW

File: dom.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  const languageList = document.createElement('ul');
+  languageList.classList.add('project-link');
+  divBox.appendChild(languageList);
+  project.language.forEach((lang) => {
+    const li = document.createElement('li');
+    li.innerHTML = lang;
+    li.classList.add('featLang');
+    languageList.appendChild(li);
+  });
+
```
Comment (brytebee) [inline dom.js:86]: - [ ] **Optional:** You have done a really good job so far. I suggest, however, that when the popup window is open, the main page should not scroll behind the popup.

- [ ] **Required:** On the desktop popup, kindly make the following changes to follow the design template:

Your Design | Template
:---: | :---:
image | image

- 1 and 2: The padding should be adjusted.
- 3 and 4: Position appropriately the button and text.
- 5 and 6: Center the modal popup on the browser window.

- [ ] **Required:** On the button, fix the font family to follow the design.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Excellent PR summary✔️
- Good commit messages✔️
- no linter errors✔️
- clean code✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dom.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+  const divPrj = document.createElement('div');
```
Comment (medaminedev66) [inline dom.js:87]: - [ ] You have built a great pop-up design. Let's make it better by fixing the height of your pop-up so your pop-up design is complete.
- Your design:
image

File: dom.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  // LANGUAGE LIST
+  const languageList = document.createElement('ul');
+  languageList.classList.add('project-link');
+  divBox.appendChild(languageList);
+  project.language.forEach((lang) => {
+    const li = document.createElement('li');
+    li.innerHTML = lang;
+    li.classList.add('featLang');
+    languageList.appendChild(li);
+  });
```
Comment (medaminedev66) [inline dom.js:85]: - [ ] In the mobile version, there is an overlap between languages and the project's image. Kindly fix this to improve your UI design.
- Your design:
image

This module is about HTML & CSS, and the Figma template is very detailed. Sticking to the design will help you practice your HTML & CSS skills better.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Ol-create , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @Ol-create  👋,

_This is an additional review as a result of the second opinion request._
Well done implementing the popup section for both mobile and desktop 👍  but you missed the languages part for the mobile view kindly fix this and submit it for another review 
image


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀! //

**Review by BenMukebo — APPROVED** [review]
Hi @Ol-create , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dom.js (hunk 1 above)
Comment (Ol-create) [reply dom.js:86]: Thank you very much @brytebee issue resolved :heavy_check_mark:

File: dom.js (hunk 3 above)
Comment (Ol-create) [reply dom.js:85]: Thank you for your time and effort @medaminedev66 :heavy_check_mark: 
I appreciate you :+1:

**Pull request closed without merging**